client/src/components/showMeMore/Observations.jsx

```diff
@@ -63,7 +63,8 @@ const Observations = props => {
       {(detailData.length ? detailData : observations)
         .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
         .map(observation => (
-          <ExpansionPanel className={classes.expansionPanel}>
+          <ExpansionPanel className={classes.expansionPanel} 
+          defaultExpanded={observations.length === 1}>
```

Comment:
I would change this to `detailData.length` instead of `observations.length`.

At least testing it on my end, `observations.length` gets the length of all observations (even ones that aren't displayed currently, whereas `detailData.length` gets the length of the the observations being shown. So even if I only made one observation today, but had observations one previous days, using `observations.length === 1` would still return false since it would be getting the length of the array of today's and the other day's observation.